setup.py

```diff
@@ -208,6 +209,9 @@ def get_spec(file_name: Path, preset: Dict[str, str], config: Dict[str, str]) ->
             elif source.startswith("class"):
                 class_name, parent_class = _get_class_info_from_source(source)
                 # check consistency with spec
+                if class_name != current_name:
+                    print('class_name', class_name, 'current_name', current_name)
```

Comment:
purposefully leaving in `print` statement?

Reply:
removed it https://github.com/ethereum/consensus-specs/pull/2937/commits/41767811234b55196d85dbc84eef74ece89ac16c